### PR TITLE
PWX-24576: Implement clouddrive Expand API in CloudOps

### PR DESCRIPTION
**Pull request by nikita-bhatia**
Jira ticket : https://portworx.atlassian.net/browse/PWX-24576

Changes done : 
Implemented Expand API for Oracle cloud

Tests done : 
Verified Expansion of disk with and without attachment to instance.

### CODE REVIEW

File: oracle/oracle.go [hunk 1]
```diff
@@ -786,3 +786,30 @@ func nodePoolContainsNode(s []containerengine.Node, e string) bool {
 	}
 	return false
 }
+
+func (o *oracleOps) Expand(volumeID string, newSizeInGiB uint64) (uint64, error) {
```
Comment (vinayakshnd) [inline oracle/oracle.go:790]: First check the current size of the disk. If requested `newSizeInGiB` is less than or equal to current size then return an error.
Take a look at how are we currently doing in AWS: https://github.com/libopenstorage/cloudops/blob/master/aws/aws.go#L937

**Review by vinayakshnd — CHANGES_REQUESTED** [review]

File: oracle/oracle.go [hunk 2]
```diff
@@ -786,3 +786,30 @@ func nodePoolContainsNode(s []containerengine.Node, e string) bool {
...
+	logrus.Println("Expand volume to size ", newSizeInGiB)
```
[the remark this answers is not in the document]
Comment (nikita-bhatia) [reply oracle/oracle.go]: Done

File: oracle/oracle.go (hunk 1 above)
Comment (nikita-bhatia) [reply oracle/oracle.go:791]: Done

**Review by vinayakshnd — APPROVED** [review]
LGTM

**Pull request merged**